### DISCUSSION TITLE
List for setting XMLHttp Request inputs & sending

### DISCUSSION THREAD

**Discussion by Alani25** [TurboWarp/packager#597] [Ideas]
I know that the [eval cloud variable](https://docs.turbowarp.org/packager/special-cloud-behaviors#eval) is already capable of sending XMLHttp requests, but I think it'd be a good feature if there was another method specifically for XHR, that is not used through a variable, but rather through a list, with every item in the list being an input (divided by a colon to separate the keys from their values), and when the last item in the list is set to "send," it's constantly replaced with the [readyState](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/readyState) of the request, then once the request is complete, another list ("XMR output") will have the outputs of the request organized (and also separated with a colon) or an error if the request failed.

This is just a suggestion that I believe will make using APIs an easier process.

**Comment by GarboMuffin** [comment]
we recommend using https://extensions.turbowarp.org/ now for your fetching needs
